docs_src/security/secrets-config-proxy.md

```diff
@@ -97,9 +97,9 @@ Proxy configuration commands (listed below) require access to the secret store m
     username=example
     password=password-from-above
 
-    vault_token=$(curl -ks "http://localhost:8200/v1/auth/userpass/login/${username}" -d "{\"password\":\"${password}\"}" | jq -r '.auth.client_token')
+    edgex-secret-store_token=$(curl -ks "http://localhost:8200/v1/auth/userpass/login/${username}" -d "{\"password\":\"${password}\"}" | jq -r '.auth.client_token')
```

Comment:
dash should not be workable as the environment variable
use `secret_store_token`

Reply:
updated